MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1034,24 +1207,67 @@ public UUID getId() {
         return id;
     }
 
-    @Nullable
-    public UUID getSpouseID() {
-        return spouse;
-    }
+    public boolean isChild() {
+        return (getAge(campaign.getCalendar()) <= 13);
+    }
+
+    //region Age Range Identification
+    // TODO : Windchild Implement Me fully - Either add or remove in Personnel Wave 2
+    // idea : have a method that allows you to determine what a person's age range would be, as this could be useful
+    // in implementing a way to display ages instead of unknown for children
+    public static final int AGE_BABY = 0;
```

Comment:
This is unimplemented (code that I'm thinking about for wave 2), but I'll add a note to myself to swap it to an enum (I personally like the integers instead of enums, but I know that's not a common view)

Reply:
I've removed the code from this PR